moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js

```diff
@@ -114,14 +128,16 @@ const ProjectSummaryKnackDataTrackerSync = ({
     body[
       process.env.REACT_APP_KNACK_DATA_TRACKER_SIGNAL_CONNECTION
     ] = signalIds;
+    
+    console.log(body);
```

Comment:
Same thing with the console.log here. Do we actually need this in the console? Or just for development?

Reply:
🙏  - thank you for catching that.